### PR TITLE
Bump supported Symfony versions

### PR DESCRIPTION
**Pull request by stloyd**
<h2>Change Log</h2> 
<div id="change-log">
  <h4>Added</h4>
  <ul id="added">
    
  </ul> 
  <h4>Fixed</h4>  
  <ul id="fixed">
    <li>Adjust Symfony bundle code to work with newer versions</li>
  </ul>
  <h4>Changed</h4>
  <ul id="changed">
    <li>Bump supported Symfony versions</li>
  </ul>  
  <h4>Updated</h4>
  <ul id="updated">
    
  </ul>    
  <h4>Removed</h4>
  <ul id="removed">
    
  </ul>
  <h4>Deprecated</h4>
  <ul id="deprecated">
    
  </ul>  
  <h4>Security</h4> 
  <ul id="security">
    
  </ul>     
</div>
<hr/>

<h2>Description</h2>

### CODE REVIEW

File: .github/workflows/test-suite.yml [hunk 1]
```diff
@@ -116,6 +116,7 @@ jobs:
         php-version:
           - "8.1"
           - "8.2"
+          - "8.3"
```
Comment (norberttech) [inline .github/workflows/test-suite.yml]: Before that, all php constraints should be set to more precise `~8.1.10 || ~8.2.0 || ~8.3.0`

Comment (stloyd) [reply .github/workflows/test-suite.yml]: Moved out to next PR.

Comment (norberttech) [reply .github/workflows/test-suite.yml]: It should be merged before or with this one since it's fixing bug with PHP version constraint

Comment (stloyd) [reply .github/workflows/test-suite.yml]: Let's merge it before this one, here it goes: https://github.com/aeon-php/aeon/pull/116

**Comment by stloyd** [comment]
@norberttech Failed test requires a restart, it crashed on bootstrapping PHP.

**Pull request merged**